core/modules/entity/entity.api.php

```diff
@@ -88,6 +89,9 @@
  *       - access callback: As in hook_menu(). 'user_access' will be assumed if
  *         no value is provided.
  *       - access arguments: As in hook_menu().
+ *     - no cache: (used by DefaultEntityController) Set to TRUE to disable
+ *         persistent caching of fully loaded entities for this bundle. Has no
```

Comment:
Minor visual thing: the second and third lines are not aligning vertically with "no cache". 

<img width="675" alt="Screen Shot 2022-05-28 at 6 04 49 PM" src="https://user-images.githubusercontent.com/9938978/170846582-2cd664e7-e6a5-42c0-b8b7-a7a6960c850f.png">

Reply:
Will be fixed in the next rev.